lib/handleRequest.js

```diff
@@ -27,7 +27,8 @@ function handleRequest (err, request, reply) {
 
   const contentType = headers['content-type']
 
-  if (method === 'POST' || method === 'PUT' || method === 'PATCH' || method === 'TRACE' || method === 'SEARCH') {
+  if (method === 'POST' || method === 'PUT' || method === 'PATCH' || method === 'TRACE' || method === 'SEARCH' ||
+    method === 'PROPFIND' || method === 'PROPPATCH' || method === 'LOCK') {
```

Comment:
And also not familiar with webdav but just pointing out that UNLOCK method wasn't added anywhere if that was unintentional